src/Servers/IIS/IIS/src/NativeMethods.cs

```diff
@@ -14,7 +14,7 @@ internal static class NativeMethods
         internal const int ERROR_NOT_FOUND = unchecked((int)0x80070490);
         internal const int ERROR_OPERATION_ABORTED = unchecked((int)0x800703E3);
         internal const int ERROR_INVALID_PARAMETER = unchecked((int)0x80070057);
-        internal const int COR_E_IO = unchecked((int)0x80131620);
```

Comment:
Do we remember what COR_E_IO was used for?

Reply:
Nope